easybuild/easyblocks/generic/rpackage.py

```diff
@@ -195,9 +175,19 @@ def install_R_package(self, cmd, inp=None):
         else:
             self.log.debug("R package %s installed succesfully" % self.name)
 
+    def update_config_guess(self, src_dir):
+        """Update any config.guess found in src_dir"""
+        for config_guess_dir in (root for root, _, files in os.walk(src_dir) if 'config.guess' in files):
+            config_guess = os.path.join(config_guess_dir, 'config.guess')
+            if not check_config_guess(config_guess):
+                updated_config_guess = obtain_config_guess(log=self.log)
+                copy_file(updated_config_guess, config_guess)
```

Comment:
This deserves some logging, like `"Replacing outdated /path/to/config.guess with more recent copy available at /path/to/new/config.guess"`

Reply:
Isn't https://github.com/easybuilders/easybuild-easyblocks/pull/1949/files#diff-aa6aaedfce9df12a4136ab4413f7971aR154 enough? But I can add it as a `debug` log entry